broker/src/models/serializers/binary_serializer.erl

```diff
@@ -1,13 +1,17 @@
 -module(binary_serializer).
 -export([load/1, dump/1]).
 
+load(null) -> undefined;
+load(undefined) -> undefined;
```

Comment:
If it's not needed, yes - let's drop it to avoid adding complexity to already kind-of-obscure parts of the app.

I guess that the `dump/1` changes are also not needed - so avoiding changes on the file at all in this PR is a strong message.